lib/global-styles.php

```diff
@@ -288,6 +288,133 @@ function gutenberg_global_styles_include_support_for_wp_variables( $allow_css, $
 	return ! ! preg_match( '/^var\(--wp-[a-zA-Z0-9\-]+\)$/', trim( $parts[1] ) );
 }
 
+/**
+ * Register webfonts defined in theme.json.
+ */
+function gutenberg_register_webfonts_from_theme_json() {
```

Comment:
We're trying to move all PHP code under the corresponding `lib/compat/wordpress-x.x` folder. Can we move these functions under `lib/compat/wordpress-6.0` (presuming that's the version we want this in core)?

Reply:
Moved 👍